src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -393,6 +462,8 @@ AutoDownloadRule AutoDownloadRule::fromVariantHash(const QVariantHash &varHash)
     rule.setAddPaused(TriStateBool(varHash.value("add_paused").toInt() - 1));
     rule.setLastMatch(varHash.value("last_match").toDateTime());
     rule.setIgnoreDays(varHash.value("ignore_days").toInt());
+    rule.setUseSmartFilter(varHash.value("smart_filter", false).toBool());
```

Comment:
We shouldn't add this code to mainstream. This is nonsense to add backward compatibility with never existing things (from the point of view of mainstream code). Those who were able to use your changes, will be able to import this in some way.